client/src/components/Resume/templates/resumePrint.css

```
@@ -0,0 +1,44 @@
+/* Print / Save as PDF — toggled via body.printing-resume */
+@media print {
+  @page {
+    margin: 12mm;
+    size: A4;
```

Stylelint (17.11.0)

[error] Expected "A4" to be "a4" (value-keyword-case)

(value-keyword-case)